package.json

```diff
@@ -42,7 +42,7 @@
       }
     },
     "test:typescript": {
-      "command": "typings-tester --dir typescript_test"
+      "command": "dtslint --expectOnly types"
```

Comment:
This is a superior testing suite to `typings-tester`. This is also the officially supported type testing module by the TypeScript team.